### PR TITLE
Bump v0.5.2

### PR DESCRIPTION
**Pull request by fewensa**
Just tag the new version. and wait for new release for darwinia-common project.

Include
- #427

### CODE REVIEW

File: thegraph/mmr/subgraph-ropsten.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: mmr
+    network: mainnet
+    source:
+      address: "0x0000000000000000000000000000000000000000"
+      abi: mmr
+      startBlock: 14288600
```
Comment (fewensa) [inline thegraph/mmr/subgraph-ropsten.yaml:11]: Please help to check `startBlock` is right? @xiaoch05

Comment (fewensa) [reply thegraph/mmr/subgraph-ropsten.yaml:11]: Also check ethereum subgraph

File: thegraph/mmr/src/mapping.ts [hunk 2]
```diff
@@ -66,36 +67,14 @@ function saveRecord(position: u64, hash: Bytes): void {
...
-    saveRecord(28573690, Bytes.fromHexString("0x969109a4eb91d9a15a53c00e2f59084bc86f5831483efddf711e31a8c7f9cac8") as Bytes);
-    saveRecord(28575737, Bytes.fromHexString("0xa9fa2096ccc970409f1405ffdda050607b3a440cb5e0bbd751ba63e171aba252") as Bytes);
-    saveRecord(28576760, Bytes.fromHexString("0xd5e9c63688124bad6eca40b522f6bc7051e37789f088437fc908f5b197462b0a") as Bytes);
-    saveRecord(28577015, Bytes.fromHexString("0x0a6597cc714e1684d31d4a0086c91d572e464fdb4104a4eae2fc4b51b59e8573") as Bytes);
-    saveRecord(28577142, Bytes.fromHexString("0xb1c9470a6869f2462ed2c703b8d9b617e9d9fe7a97af3929e751a2c77850df14") as Bytes);
-    saveRecord(28577173, Bytes.fromHexString("0x4d99a90b98432aeb87ff169cd2160654cb21172918d99168605dfd35560bcc44") as Bytes);
-    saveRecord(28577188, Bytes.fromHexString("0xd4a62a7fe579348076e2395ff7c7b7c257fc2a69bdc2c818eed86396e4801547") as Bytes);
+  const minit = mmrInit[chain.chain];
+  const records = minit.records;
+  for (const item of records) {
```
Comment (fewensa) [inline thegraph/mmr/src/mapping.ts:73]: @xiaoch05 Please check there

**Comment by fewensa** [comment]
About wrong Ropsten chain hash index by theGraph,  please follow https://github.com/graphprotocol/indexer/issues/379

**Comment by fewensa** [comment]
https://github.com/darwinia-network/bridger/pull/424/commits/33aa26600a01c29b957bac63e0f369f689a68c89

fix the wrong condition for #286 

https://github.com/darwinia-network/bridger/pull/286/files#diff-c1bbb0a9e23066475d24431617d8515ca11d65e3a4bf7546e36f532b1b670ec2R11

**Review by xiaoch05 — APPROVED** [review]

**Review by HackFisher — CHANGES_REQUESTED** [review]
Hold on and let's do the upgration first.

**Pull request merged**